Source/DafnyLanguageServer/DafnyLanguageServer.csproj

```diff
@@ -8,6 +8,7 @@
     <RootNamespace>Microsoft.Dafny.LanguageServer</RootNamespace>
     <OutputPath>..\..\Binaries\</OutputPath>
     <ValidateExecutableReferencesMatchSelfContained>false</ValidateExecutableReferencesMatchSelfContained>
+    <PackageVersion>3.6.0.40511</PackageVersion>
```

Comment:
What's the use of publishing this as a package?

Reply:
Do you know whether it's possible to set it up so that the language server gets distributed along with the main CLI? And, if so, how to do it? I think that'd be ideal, but wasn't sure if it would be possible, at least without significant refactoring.

Reply:
I'm sure it possible, and I think we should, but we should spend a little more time discussing whether we want it. For now, maybe just not publish DafnyLanguageServer because unless you see a reason why it would be useful to consume for other .NET projects.

Reply:
Sounds reasonable.